Source/DafnyCore/Rewriters/RefinementTransformer.cs

```diff
@@ -58,7 +58,8 @@ public RefinementToken(IToken tok, ModuleDefinition m)
       }
 
       var rtok = tok as RefinementToken;
-      return rtok != null && rtok.InheritingModule == m;
+      var result = rtok != null && rtok.InheritingModule == m;
+      return result;
```

Comment:
Note: when you are debugging, you can add a breakpoint on the return statement and step through. The debugger will display the returned value. That way, you don't accidentally commit temporary variable names

Reply:
Thanks